functions.php

```diff
@@ -9,6 +9,17 @@
  * @since Twenty Twenty-Five 1.0
  */
 
+/**
+ * Add theme support for post formats.
+ *
+ * @since Twenty Twenty-Five 1.0
+ * @return void
+ */
+function twentytwentyfive_post_format_setup() {
```

Comment:
Let's wrap this in an `if ( ! function_exists(...` like the other functions in the file.